src/sentry/tasks/post_process.py

```diff
@@ -211,7 +211,7 @@ def post_process_group(event, is_new, is_regression, is_new_group_environment, *
 
             for plugin in plugins.for_project(event.project):
                 plugin_post_process_group(
-                    plugin_slug=plugin.slug, event=event, is_new=is_new, is_regresion=is_regression
+                    plugin_slug=plugin.slug, event=event, is_new=is_new, is_regression=is_regression
```

Comment:
Hmm, this is a task and we typically don't change param names on those, but looks like we don't ever actually call it as a task anymore. afaict this parameter is unused in all plugins anyway, so should be safe (and could potentially just be removed)